<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>421</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>8</y>
     <width>731</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="settings_label">
      <property name="font">
       <font>
        <pointsize>32</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="language_label">
      <property name="text">
       <string>Language</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="russian_radio">
      <property name="text">
       <string>Russian (Русский)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="english_radio">
      <property name="text">
       <string>English</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="default_delay_label">
        <property name="text">
         <string>Default typing delay</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="default_delay_spinbox"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="dynamic_refresh_checkbox">
      <property name="text">
       <string>Dynamic Refresh (May hurt perfomance)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="auto_compatibility_checkbox">
      <property name="text">
       <string>Auto launch playback in compatibility mode when possible</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="stop_unexpected_playback_checkbox">
      <property name="text">
       <string>Stop playback in unexpected situation</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="save_btn">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>SAVE</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
